README.md

```diff
@@ -1,16 +1,20 @@
-# 💊 ReVanced Manager
+<p align="center">
+  <img width="800" src="Logo.png">
+</p>
 
-The official ReVanced Manager based on Flutter.
 
-> **Warning**: This repository currently has no active maintainer. For that reason, development is stale. Active development continues for [revanced-cli](https://github.com/revanced/revanced-cli). If you are interested in maintaining this repository, please let us know at manager@revanced.app.
+### <p align="center">  The official ReVanced Manager based on Flutter</p>
+
+> **Warning**: This repository currently has no active maintainer. For this reason, development is stale. Active development continues for [revanced-cli](https://github.com/revanced/revanced-cli). If you are interested in maintaining this repository, please let us know at manager@revanced.app.
 
 ## 🔽 Download
-To download the Alpha version of Manager, go [here](https://github.com/revanced/revanced-manager/releases/latest) and install the provided APK file.
+ReVanced Manager is still in early development (ALPHA) . You can install the latest version [here](https://github.com/revanced/revanced-manager/releases/latest).
 
 ## 📝 Prerequisites
-1. Android 8 or higher
-2. Does not work on some armv7 devices
-3. [Vanced MicroG](https://github.com/TeamVanced/VancedMicroG/releases) required for YouTube and YouTube Music (Only for non-root)
+1. Android 8 or higher.
+2. [Vanced MicroG](https://github.com/TeamVanced/VancedMicroG/releases) is required for YouTube and YouTube Music (only for non-root).
+
+&nbsp; 🟥 Does not work on some ARMv7 devices
```

Comment:
This is inconsistent markdown. Use lists.

Reply:
Not working on ARMv7 devices is not a prerequisite so I thought that including it in the list doesn't make any sense

Reply:
In that case it doesn't belong in the section at all and should be moved outside